contentsrc/mapsrc/ASI-Jac2-Deima.vmf

```diff
@@ -164083,10 +164083,10 @@ entity
 {
 	"id" "1100771"
 	"classname" "trigger_multiple"
-	"origin" "8104 6872 -8"
+	"origin" "8472 6872 -8"
 	"spawnflags" "16"
 	"StartDisabled" "0"
-	"wait" "1"
+	"wait" "0.1"
```

Comment:
OnStartTouch isn't affected by the wait timer, so this should probably be reverted to 1 second